game-core/src/main/java/games/strategy/engine/history/change/HistoryChangeFactory.java

```diff
@@ -0,0 +1,28 @@
+package games.strategy.engine.history.change;
+
+import games.strategy.engine.data.Territory;
+import games.strategy.engine.data.Unit;
+import games.strategy.engine.history.change.units.RemoveUnits;
+import games.strategy.engine.history.change.units.TransformDamagedUnits;
+import java.util.Collection;
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class HistoryChangeFactory {
```

Comment:
What other history change objects do you foresee? How many might we have total and will a single factory for them all scale well?

Reply:
I patterned this off of the existing `ChangeFactory`.  I could imagine one history change class per type of line in the history.

Reply:
Okay. If it grows to be overwhelming we can probably deal with it.